Overview.bs

```diff
@@ -400,46 +401,43 @@ Note: The user agent is not required to run animations on every frame. It is leg
       provide a different service level according to their policy.
 
 
-When the user agent wants to <dfn>run animators</dfn>, it <em>must</em> iterates over <a>animator
-instance list</a> as |instance|:
+When the user agent wants to <dfn>run animators</dfn>, it <em>must</em> iterate over the values
+in the <a>(animator name, root element) to instance map</a> as |instance|, performing the following
+steps for each |instance|:
+
+  1. If |instance| is <b>null</b>, abort the following steps.
 
-  1. If the <a>animation requested flag</a> for the instance is <a>frame-current</a>
+  2. If the <a>animation requested flag</a> for the instance is <a>frame-current</a>
         or the proxies belonging to the |instance| will not be visible within the visual viewport
         of the current frame the user agent <em>may</em> abort all the following steps.
 
         Issue: Consider giving user agents permission to skip running animator instances to throttle
         slow animators.
 
-  2. Let |name| be the <a>animator name</a> of |instance|.
+  3. Let |name| be the <a>animator name</a> of |instance|.
 
-  3. Let |workletGlobalScope| be a {{AnimationWorkletGlobalScope}} from the list of <a>worklet's
+  4. Let |workletGlobalScope| be a {{AnimationWorkletGlobalScope}} from the list of <a>worklet's
         WorkletGlobalScopes</a> from the animation {{Worklet}}.
 
-  4. Let the |definition| be the result of looking up |name| on the |workletGlobalScope|'s
+  5. Let the |definition| be the result of looking up |name| on the |workletGlobalScope|'s
         <a>animator name to animator definition map</a>.
 
       If |definition| does not exist abort the following steps.
 
-  5. Let |animatorInstance| be the result of looking up |name| on |workletGlobalScope|'s
-        <a>animator name to instance map</a>. If |animatorInstance| is null run the following
-        substeps:
-
-        Issue: Todo: Add steps to create new animatorInstance given the animator definition.
```

Comment:
This might be part of creating an element proxy. i.e. Identify the root element, and if no instance exists for that name, root element, then create one.